src/parallelograms-home.pug

```diff
@@ -0,0 +1,34 @@
+extends templates/layout
```

Comment:
I think the content of this file should be in `index.pug`, in the `div(v-if="user")` section (rather than in a separate template).